c2_components/Android.mk

```diff
@@ -25,15 +25,21 @@ define build_components
     libhardware libdl liblog libcutils\
     $(MFX_C2_SHARED_LIBS) \
         libexpat \
+        libsync \
+        libdrm \
+        libutils \
+        libhidlbase \
         libgralloctypes \
         libstagefright_foundation \
         android.hardware.media.bufferpool@2.0 \
         android.hardware.graphics.bufferqueue@2.0 \
         android.hardware.graphics.common@1.2 \
+        android.hardware.graphics.mapper@4.0 \
         libsfplugin_ccodec_utils
 
   LOCAL_STATIC_LIBRARIES := \
-    libmfx_c2_buffers
+    libmfx_c2_buffers \
+    libminigbm_cros_gralloc_celadon
```

Comment:
Why we include this libminigbm_cros_gralloc_celadon static libs here?

Reply:
```
ld.lld: error: undefined symbol: cros_gralloc_convert_handle(native_handle const*)
ld.lld: error: undefined symbol: i915_private_invert_format(int)
```
These 2 function calling need include this lib. Let me see if there is another way to avoid including it.